packages/integrations/useNProgress/index.md

```diff
@@ -0,0 +1,47 @@
+# useNProgress
+
+> Reactive wrapper for [`nprogress`](https://github.com/rstacruz/nprogress)
+
+## Usage
+
+```js {6}
+import { useNProgress } from '@vueuse/integrations'
+
+const { isLoading } = useNProgress()
+
+function toggle() {
+  isLoading.value = !isLoading.value
+}
+```
+
+### Passing a source ref
+
+You can pass a `ref` to it, changes from of the source ref will be reflected to your loading progress.
+
+```js {4}
+import { ref } from 'vue'
+import { useNProgress } from '@vueuse/integrations'
+
+const isLoading = ref(false)
```

Comment:
Do we need to update the docs? I think we are accepting numbers now, or do you think we should support boolean as well?

Reply:
no, I don't think we need to support boolean here